### PR TITLE
[DOCS] Update readme (unify testing, ci, monitoring)

### PR DESCRIPTION
**Pull request by shir22**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,316 +11,264 @@
...
+   </summary>
+
+- Each check tests for a specific potential problem. Deepchecks has many preimplemented checks for
+  finding issues with the model's performance (e.g. identifying weak segments), 
+  data distribution (e.g. detect drifts or leakages) and data integrity (e.g. find conflicting labels).
+- Customizable: each check has many configurable parameters, and custom checks can easily be implemented.
+- Can be run manually (during research) or triggered automatically (in CI processes or production monitoring)
+- Check results can be consumed by: 
+   - [Saving to HTML](https://docs.deepchecks.com/stable/general/usage/export_save_results.html) or [viewing](https://docs.deepchecks.com/stable/general/usage/showing_results.html) in Jupyter - for visual output for human analysis
+   - [JSON output](https://docs.deepchecks.com/stable/general/usage/export_save_results.html) - for processing with code
```
Comment (noamzbr) [inline README.md:117]: Programmatic output (result.value) is the best option if you're working with deepchecks from python.

File: README.md [hunk 2]
```diff
@@ -11,316 +11,264 @@
...
+
+This repo is our main repo as all components use the deepchecks checks in their core. See the [Getting Started](#--getting-started) section for more information about installation and quickstarts for each of the components.
+If you want to see deepchecks monitoring's code, you can check out the 
+files in the [deepchecks/monitoring](https://github.com/deepchecks/monitoring) repo.
+
+
+<details close>
+   <summary>
+      <h2>
+         ✅ Deepchecks Checks
```
[the remark this answers is not in the document]
Comment (noamzbr) [reply README.md:106]: Also the package we're talking about in this part is actually "deepchecks" lower caps...

File: README.md [hunk 3]
```diff
@@ -11,316 +11,264 @@
...
+      </h3>
+   </summary>
+
+#### Deepchecks Testing (and CI) Installation
 
 ```bash
 pip install deepchecks -U --user
 ```
 
+For installing also the nlp submodle, replace ``deepchecks`` with ``"deepchecks[vision]"``
```
Comment (noamzbr) [inline README.md:149]:
```suggestion
For installing also the nlp submodle, replace ``deepchecks`` with ``"deepchecks[nlp]"``
```

File: README.md [hunk 4]
```diff
@@ -11,316 +11,264 @@
...
+and for installing it with the computer vision submodule, replace it with ``"deepchecks[nlp]"``.
```
Comment (noamzbr) [inline README.md:150]:
```suggestion
and for installing it with the computer vision submodule, replace it with ``"deepchecks[vision]"``. 
```

File: README.md (hunk 3 above)
Comment (noamzbr) [reply README.md:149]: Maybe note also the nlp-properties thing? Not a must

Comment (noamzbr) [inline README.md:149]: This maybe should be a bullet list?

File: README.md [hunk 5]
```diff
@@ -11,316 +11,264 @@
...
-> **Recommended - download the code and run it locally** on the built-in dataset and (optional) model, or **replace them with your own**.
-
-
-### 🚀 See Our Checks Demo
-
-Play with some of the existing checks in our [Interactive Checks Demo](
-   https://checks-demo.deepchecks.com/?check=No+check+selected&utm_source=github.com&utm_medium=referral&utm_campaign=readme&utm_content=try_it_out), 
-and see how they work on various datasets with custom corruptions injected.
+If you're using deepchecks also for production monitoring,
+you can deploy a hobby instance in one line on Linux/MacOS (Windows is WIP!) with Docker:
```
Comment (noamzbr) [inline README.md:158]: what is a hobby instance?

File: README.md [hunk 6]
```diff
@@ -11,316 +11,264 @@
...
 
+```
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/deepchecks/monitoring/main/deploy/deploy-oss.sh)"
+```
+This will automatically download the necessary dependencies and start the application locally.
+Installing our open source service is an excellent way for local usage, note however,
```
Comment (noamzbr) [inline README.md:164]:
```suggestion
Installing our open source service is an excellent way to try monitoring locally. Note however, 
```

File: README.md [hunk 7]
```diff
@@ -11,316 +11,264 @@
...
-```
+<details open>
+   <summary>
+      <h4>
+         Deepchecks Testing Quickstart
+      </h4>
+   </summary>
 
-Which will result in a report that looks like this:
+Jump straight over to the respective quickstart docs:
```
Comment (noamzbr) [inline README.md:181]:
```suggestion
Jump right into the respective quickstart docs:
```

File: README.md [hunk 8]
```diff
@@ -11,316 +11,264 @@
...
+- [Tabular Quickstarts](https://docs.deepchecks.com/stable/tabular/auto_tutorials/quickstarts/index.html)
+- [NLP Quickstarts](https://docs.deepchecks.com/stable/nlp/auto_tutorials/quickstarts/index.html)
+- [Vision Quickstarts](https://docs.deepchecks.com/stable/vision/auto_tutorials/quickstarts/index.html) 
+to have it up and running on your data.
```
Comment (noamzbr) [inline README.md:185]: This is appended to the vision line

File: README.md [hunk 9]
```diff
@@ -11,316 +11,264 @@
...
 
+You'll then be able to view all the chosen checks checks and inspect their status and results,
```
Comment (noamzbr) [inline README.md:187]: Not clear when is "then"

File: README.md [hunk 10]
```diff
@@ -11,316 +11,264 @@
...
+- [Tabular Quickstarts](https://docs.deepchecks.com/stable/tabular/auto_tutorials/quickstarts/index.html)
+- [NLP Quickstarts](https://docs.deepchecks.com/stable/nlp/auto_tutorials/quickstarts/index.html)
+- [Vision Quickstarts](https://docs.deepchecks.com/stable/vision/auto_tutorials/quickstarts/index.html) 
+to have it up and running on your data.
 
+You'll then be able to view all the chosen checks checks and inspect their status and results, 
+with the visual output that looks like this:
 
 <p align="center">
    <img src="docs/source/_static/images/general/model_evaluation_suite.gif" width="800">
```
Comment (noamzbr) [inline README.md:191]: I know it's not new, but now that I'm looking at it, it looks low-res...

File: README.md [hunk 11]
```diff
@@ -11,316 +11,264 @@
...
->   <p align="left">
->      <img src="docs/source/_static/images/general/train-test-drift-output.png">
->   </p>
-
-
-## 🙋🏼  When Should You Use Deepchecks?
-
-While you’re in the research phase, and want to validate your data, find potential methodological problems, 
-and/or validate your model and evaluate it.
+Jump straight over to the
```
Comment (noamzbr) [inline README.md:203]: I have an issue with "straight over" somehow :P

File: README.md [hunk 12]
```diff
@@ -11,316 +11,264 @@
...
+Jump straight over to the 
+[monitoring quickstart docs](https://docs.deepchecks.com/monitoring/stable/user-guide/tabular/auto_quickstarts/plot_quickstart.html)
+to have it up and running on your data.
 
+You'll then be able to see the checks results over time, set alerts, and interact
+with the dynamic deepchecks UI that looks like this:
 
 <p align="center">
-   <img src="/docs/source/_static/images/general/pipeline_when_to_validate.svg">
+   <img src="docs/source/_static/images/general/monitoring-app-ui.gif" width="800">
```
Comment (noamzbr) [inline README.md:211]: Not rendered in git, dunno if it works when building docs.

Comment (noamzbr) [reply README.md:211]: Previous gif did render so a bit suspicious

File: README.md [hunk 13]
```diff
@@ -11,316 +11,264 @@
...
--  The model's training data with labels
--  Test data (which the model isn't exposed to) with labels
--  A [supported
-    model](https://docs.deepchecks.com/stable/user-guide/supported_models.html?utm_source=github.com&utm_medium=referral&utm_campaign=readme&utm_content=running_a_suite) (e.g. scikit-learn models, XGBoost, any model implementing the ``predict`` method in the required format)
+</details>
 
+## 📜 Open Source vs Paid 
 
-### Supported Data Types
+Deepchecks' projects (``deepchecks/deepchecks`` & ``deepchecks/monitoring``) are open source and are released under [AGPL 3.0](./LICENSE).
```
Comment (noamzbr) [inline README.md:237]: Make it clearer that THIS current repo (``deepchecks/deepchecks``) is 100% AGPL free forever.

File: README.md [hunk 14]
```diff
@@ -11,316 +11,264 @@
...
+
+At its core, Deepchecks includes a wide variety of built-in Checks,
+for testing all types of data and model related issues.
+These checks are implemented fo various models and data types (Tabular, NLP, Vision), 
+and can easily be customized and expanded. 
+
+The check results can be used for automatically making informed decisions
+regarding your model's production-readiness, and for monitoring it over time when in production.
+They can be examined with visual reports (by saving them to an HTML file, or seeing them in Jupyter),
+processed with code (using their json output), and inspected and collaborated upon with a Deepchecks' dynamic UI
```
[the remark this answers is not in the document]
Comment (shir22) [reply README.md:71]: just wanted to make it clear that it's not just a "prettier than the plotly + widgets UI" and static
but rather that it's something that's allows further inspection / investigation, and doing some active stuff...
Hinting that it enables RCA also without code etc.
Do you think that it's better to drop it / have a suggestion for a better (and still quite short and concise) phrasing that'll hint that?

File: README.md [hunk 15]
```diff
@@ -11,316 +11,264 @@
...
+These checks are implemented fo various models and data types (Tabular, NLP, Vision), 
+and can easily be customized and expanded. 
+
+The check results can be used for automatically making informed decisions
+regarding your model's production-readiness, and for monitoring it over time when in production.
+They can be examined with visual reports (by saving them to an HTML file, or seeing them in Jupyter),
+processed with code (using their json output), and inspected and collaborated upon with a Deepchecks' dynamic UI 
+(for examining test results and for production monitoring).
+
+<!---
```
[the remark this answers is not in the document]
Comment (shir22) [reply README.md:74]: no real reason, commented out just in case I'd receive comment to elaborate that part
(and because the commit history will be squashed)

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (shir22) [reply README.md:106]: How about the new title?

File: README.md (hunk 5 above)
Comment (shir22) [reply README.md:158]: hinting that it's not "production grade" and if you need something that's scalable, and you care about production-related metrics (e.g. downtime) then you should consider to use the managed/licensed deployment

Comment (shir22) [reply README.md:158]: phrasing inspired by posthog, but I think it's used widely enough

File: README.md (hunk 11 above)
Comment (shir22) [reply README.md:203]: :)

File: README.md (hunk 12 above)
Comment (shir22) [reply README.md:211]: try refreshing? (maybe internet connection cause it works for me)

File: README.md (hunk 10 above)
Comment (shir22) [reply README.md:191]: I think I played a bit around with the gif configurations and chose that eventually, don't remember the exact considerations... though personally I think there are some advantages to it - it give the feeling of a high-level (visual) overview and it's ok that you can't really dive into the text, anyway it's not really possible. Also it looks more gif-y and less like a video which is nice.
Do you think worth redoing? (possible but if so I'll do it in a future PR after the launch. and then need to fix also the monitoring gif which is even more low-res).

File: README.md (hunk 13 above)
Comment (shir22) [reply README.md:237]: On purpose this readme doesn't distinct between the repos so it may be confusing to relate to that.
Did make the phrasing in the next sentence a bit direct.

Also wouldn't want to commit on that, in case we add in the future regulation-related checks that need a license (and are likely to be in deepchecks/deepchecks)

File: README.md (hunk 5 above)
Comment (noamzbr) [reply README.md:127]: I was a bit baffled by it, maybe "free tier" or something?

**Review by noamzbr — APPROVED** [review]

File: README.md (hunk 5 above)
Comment (shir22) [reply README.md:127]: updated :) thanks
